src/main/kotlin/com/chutneytesting/kotlin/dsl/ChutneyStepImplExtensions.kt

```diff
@@ -581,12 +581,16 @@ fun ChutneyStepBuilder.JmsBrokerStartTask(
 fun ChutneyStepBuilder.SqlTask(
     target: String,
     statements: List<String>,
-    outputs: Map<String, Any> = mapOf()
+    outputs: Map<String, Any> = mapOf(),
+    nbLoggedRow: Integer
 ) {
     implementation = ChutneyStepImpl(
         type = "sql",
         target = target,
-        inputs = mapOf("statements" to statements),
+        inputs = mapOf(
+            "statements" to statements,
+            "nbLoggedRow" to nbLoggedRow
```

Comment:
It isn't a required input.
Maybe something like this : `("nbLoggedRow" to nbLoggedRow).takeIf { nbLoggedRow.isNotBlank() }`